payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -356,6 +356,13 @@ constructor(
             isVoucher = false,
             requiresMandate = true,
             hasDelayedSettlement = true
+        ),
+        CashAppPay(
+            code = "cashapp",
+            isReusable = false,
+            isVoucher = false,
+            requiresMandate = false,
```

Comment:
I’m a bit worried about this property, which is used in `Confirm***IntentParams.create()`. Cash App Pay requires a mandate for setup intents and payment intents with SFU, so this constant value is incorrect.

Since `PaymentMethod.Type` is part of the public API, changing this property is cumbersome. I could add a private function `requiresMandateForIntent(clientSecret)` (or something with a smoother name), but this wouldn’t work for PI+SFU.

Reply:
Let's talk offline about this.

Reply:
What is the resolution for this?